### ISSUE TITLE
`customizeExcelCell` in pivot grid

### ISSUE THREAD

**Issue by Semigradsky** [DevExpress/DevExtreme#12003]
We have `customizeExcelCell` function for pivot grid but we can't customize because we don't know what is value passing to `customizeExcelCell`.

Would be nice to have `gridCell` in arguments (like for dxDataGrid) or smth like this for determining what is value currently customizing.

**Comment by Semigradsky** [comment]
Fox example I want to set different number format depends on column name/type.

**Comment by artem-kurchenko** [comment]
`customizeExcelCell` is not a part of the built-in PivotGrid API in v19.2

@EugeniyKiyashko  @IgnatovDan  what do you think of this as a possible improvement?

**Comment by IgnatovDan** [comment]
We researched how to add the customizeExcelCell event to PivotGrid and found that it will take us some time to implement/test/describe/demonstrate a possible approach.
While researching the issue, we found several scenarios that are not supported by our current DataGrid/PivotGrid exporting code and we cannot improve our code within a reasonable time frame (see [Excel Export Enhancements](https://github.com/DevExpress/DevExtreme/issues/7294)).
We investigated these scenarios and implemented a new solution to export DataGrid to XLSX, see [DevExtreme - Data Grid - New Excel Export API Update (CTP in v19.1)](https://community.devexpress.com/blogs/javascript/archive/2019/10/04/devextreme-data-grid-new-excel-export-api-update-ctp-in-v19-1.aspx).
This approach allows you to implement much more scenarios and we plan to introduce a similar solution for PivotGrid in one of our next releases (though I cannot promise any strict time frame).
